tests/utils/shippable/units.sh

```diff
@@ -22,6 +22,11 @@ esac
 
 ansible-test env --timeout "${timeout}" --color -v
 
+if [ "$2" == "2.10" ]; then
```

Comment:
Despite we seeing the problem in 2.10, I wonder if it wouldn't be simpler to simply add the two python-gitlab lines to the requirements, as devised below, regardless of the ansible version.

Reply:
That would prevent the newer python-gitlab version to be used for the tests with all newer Ansible versions. That would be a really bad idea IMO :)

Reply:
Ok, makes sense. LGTM to me.